webrender/src/picture.rs

```diff
@@ -1061,6 +1066,29 @@ impl TileCache {
             // Decide how to handle this tile when drawing this frame.
             if tile.is_valid {
                 self.tiles_to_draw.push(TileIndex(i));
+
+                #[cfg(feature = "debug_renderer")]
```

Comment:
I think you can put the `#[cfg()]` just on top of the `if` and deindent it, here and below.

Reply:
Unfortunately that gives ```error: attributes are not yet allowed on `if` expressions```